src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -430,9 +430,11 @@ senderBalance < (UInt256)transaction.GasLimit * transaction.MaxFeePerGas + value
                         _stateProvider.CreateAccount(gasBeneficiary, fees);
                     }
 
+                    UInt256 burntFees = UInt256.Zero;
```

Comment:
This logic seems to be incorrect when Eip1559FeeCollector is null.
If we don't have Eip1559FeeCollector burntFees should be still reported. For mainnet we don't have Eip1559FeeCollector.

Reply:
they are reported few lines lower

Reply:
Do we have tests with Eip1559FeeCollector and without Eip1559FeeCollector?